Extend IPFS introduction (#607)
orcestra-campaign/book: orcestra_book/ipfs.md

## IPFS@ORCESTRA

The latest version of the dataset is available via:

Subsystem | Name/Path
--- | ---
[IPNS](https://docs.ipfs.tech/concepts/ipns/) | `/ipns/latest.orcestra-campaign.org`
HTTP gateway | https://latest.orcestra-campaign.org

You can find an overview presentation about the usage of IPFS during ORCESTRA [here](https://orcestra-campaign.github.io/ipfs_intro/#/title-slide).

## Accessing data with IPFS

To obtain data from the IPFS network, you need access to an [IPFS Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway).

::::{grid} 1 1 2 2
:::{grid-item-card} Local machine
We recommend that you use your own local gateway for faster access and local caching.
The simplest solution is to install [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/), which provides a graphical user interface and runs a [Kubo daemon](https://docs.ipfs.tech/install/command-line/) in the background.

> If you are unable to install software on your machine (e.g. work laptop), you can configure IPFS to use the public HTTPS gateway by setting:

Reviewer: ```suggestion
If you are **unable** to install software on your machine (e.g. work laptop), you can configure IPFS to use the public HTTPS gateway by setting:
```

```
export IPFS_GATEWAY=https://ipfs.io
```
:::

:::{grid-item-card} High-performance Computer

> You should check, whether your HPC centre provides an IPFS gateway (i.e. whether `IPFS_GATEWAY` is set).

Reviewer: ```suggestion
You should check, whether your HPC centre provides an IPFS gateway (e.g. whether `IPFS_GATEWAY` is set).
```

> If this is not the case, you can configure IPFS to use the public HTTPS gateway by setting:

Reviewer: ```suggestion
If this is **not** the case, you can configure IPFS to use the public HTTPS gateway by setting:
```

```
export IPFS_GATEWAY=https://ipfs.io
```
:::
::::

> If you want to access the ORCESTRA data using Python, you will need to install the [`ipfsspec>=0.5.3`](http://pypi.org/project/ipfsspec/) package.

Reviewer: ```suggestion
If you want to access the ORCESTRA data using Python, you will need to install the [`ipfsspec>=0.6.0`](http://pypi.org/project/ipfsspec/) package.
```

> It is highly recommended to install `ipfsspec` using pip, the version provided via `conda-forge` is outdated.

Reviewer: ```suggestion
It is essential to install `ipfsspec` using pip, the version provided via `conda-forge` is outdated and **broken**.
```
